components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -48,27 +49,27 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient has been uploaded to this server
+phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
 phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
 phenotips.PushPatient.patientUploadedDays=days ago
 phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=as
-phenotips.PushPatient.patientUploadedUpdate=Further pushes will attempt to update the patient on the remote server.
+phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
```

Comment:
Or you can bundle a small spritf JavaScript implementation, so we don't
have regex all over the place.
On Mar 22, 2016 1:09 AM, "Sergiu Dumitriu" <notifications@github.com> wrote:

> In
> components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties
> <https://github.com/phenotips/phenotips/pull/1974#discussion_r56938053>:
>
> >  phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
> >  phenotips.PushPatient.patientUploadedDays=days ago
> >  phenotips.PushPatient.patientUploadedHours=hours ago
> > -phenotips.PushPatient.patientUploadedAsID=as
> > -phenotips.PushPatient.patientUploadedUpdate=Further pushes will attempt to update the patient on the remote server.
> > +phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
> > +phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
>
> Just like you replace __serverID__, you would replace {0} instead.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/phenotips/phenotips/pull/1974/files/f046cc9970c3e51cc9fc8e8206ee036374003a39#r56938053>
>